### PR TITLE
From spcm

### PR DESCRIPTION
**Pull request by p770193**
Resolves issue #678 .

### CODE REVIEW

File: WrightTools/data/_spcm.py [hunk 1]
```diff
@@ -23,8 +23,9 @@
 # --- from function -------------------------------------------------------------------------------
 
 
-def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, verbose=True):
-    """Create a data object from Becker & Hickl `spcm`__ software.
+def from_spcm(filepath, name=None, *, delimiter=",", parent=None, verbose=True):
+    """Create a ``Data`` object from a Becker & Hickl `spcm`__ file (ASCII-exported, ``.asc``).
+    If provided, setup parameters are stored in the ``attrs`` dictionary of the ``Data`` object.
```
Comment (ksunden) [inline WrightTools/data/_spcm.py]: Need a new line (for docstrings to be properly rendered on ReadTheDocs

File: WrightTools/data/_spcm.py [hunk 2]
```diff
@@ -37,9 +38,6 @@ def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, v
         Default is None.
     delimiter : string (optional)
         The string used to separate values. Default is ','.
-    format : {'ascii'} (optional)
-        Force file to be interpreted as a specific format. Default is None
-        (autorecognized).
```
Comment (ksunden) [inline WrightTools/data/_spcm.py]: Was format used?, I didn't see it, but just checking to be sure

File: WrightTools/data/_spcm.py [hunk 3]
```diff
@@ -74,19 +68,61 @@ def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, v
...
+                    splitted = line.split()
+                    value = splitted[-1][1:-1].split(",")
+                    key = " ".join(splitted[:-1])
+                    headers[key] = value
+            line = f.readline().strip()
+            if "END" in line:
+                header_lines += 1
+                break
+    # initialize data object
+    kwargs = {"name": name, "kind": "spcm", "source": filepath, **headers}
```
Comment (ksunden) [inline WrightTools/data/_spcm.py]: If we have access to the time of collection, it is intended behavior that the data object "created" date use that time. See `_pycmds.py` for an example of such. (Just a matter of passing in the kwarg/ensuring the date format is what we want) 

Here, it seems to be the combination of two header fields "Date" and "Time"

File: WrightTools/data/_spcm.py (hunk 2 above)
Comment (p770193) [reply WrightTools/data/_spcm.py]: format is not used

File: WrightTools/data/_spcm.py [hunk 4]
```diff
@@ -74,27 +70,54 @@ def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, v
...
+                    headers[key] = value
+            line = f.readline().strip()
+            if "END" in line:
+                header_lines += 1
+                break
+    if "Date" in headers.keys() and "Time" in headers.keys():
+        # NOTE:  reports created in local time, no-way to calculate absolute time
+        created = " ".join([headers["Date"], headers["Time"]])
+        created = time.strptime(created, "%Y-%m-%d %H:%M:%S")  
+        created = timestamp.TimeStamp(time.mktime(created)).RFC3339
```
Comment (ksunden) [inline WrightTools/data/_spcm.py]: You get the created time here, but looks like you never do anything with it should be added to the kwargs as the "special/reserved" attribute `"created"`

File: WrightTools/data/_spcm.py (hunk 3 above)
Comment (darienmorrow) [reply WrightTools/data/_spcm.py]: What timezone would the investment be in?

**Pull request merged**